main.js

```diff
@@ -11,31 +11,41 @@ function generateNumbers() {
     return allNumbers;
 }
 
-function displayNumbers(allNumbers) {
-    let numbersOnScreen = allNumbers.length;
-
+function displayNumbers() {
+    const randomNumbers = generateNumbers();
+    const sortedNumbers = randomNumbers.slice(0).sort();
+    let numbersOnScreen = randomNumbers.length;
      // using a for loop instead of forEach as the array.pop changes the length over each iteration
      for (let i = 0; i < numbersOnScreen; i++) {
-        let randNumber = allNumbers.pop();
+        let randNumber = randomNumbers.pop();
         let number = document.createElement('div');
 
         number.textContent = randNumber;
         number.setAttribute('class', 'numberBox');
 
          document.querySelector('#play_area').appendChild(number);
     }
-
+    let selectedOrder = [];
     document.querySelectorAll('.numberBox').forEach((element) => {
         element.addEventListener('click', () => {
+
```

Comment:
Is it maybe worth it for eligibility to add all the logic here within a function and then call it here instead?

Reply:
legibilaty?

Reply:
encapsulation too

Reply:
I don't see the benefit to that?